.travis.yml

```diff
@@ -1,6 +1,5 @@
 language: ruby
 rvm:
-  - jruby-1.7.27
   - jruby-9.0.5.0
```

Comment:
Do we want to run tests for `jruby-9.0.5.0`?

That JRuby represents Ruby 2.2.3, and has been marked `:buggy` by the `ruby_dep` system.